app/src/main/java/com/x8bit/bitwarden/data/platform/datasource/network/retrofit/Retrofits.kt

```diff
@@ -36,11 +37,14 @@ interface Retrofits {
     val unauthenticatedIdentityRetrofit: Retrofit
 
     /**
-     * Allows access to static API calls (ex: external APIs) that do not therefore require
-     * authentication with Bitwarden's servers.
+     * Allows access to static API calls (ex: external APIs).
      *
-     * No base URL is supplied as part of the builder and no longer is added to make this URL
-     * dynamically updatable.
+     * @param isAuthenticated Indicates if the [Retrofit] instance should use authentication.
+     * @param baseUrl The static base url associated with this retrofit instance. This can be
+     * overridden with the [Url] annotation.
      */
-    val staticRetrofitBuilder: Retrofit.Builder
+    fun createStaticRetrofit(
+        isAuthenticated: Boolean = false,
+        baseUrl: String = "https://www.bitwarden.com",
```

Comment:
Not that it matters too much, but do we want to make this `https://api.bitwarden.com` to match what we're using as the default base URL elsewhere?

Reply:
Sure, I went with `https://www.bitwarden.com` since that is what all the existing places use. But it will be removed overridden in those cases anyways